shared/styles/css.d.ts

```diff
@@ -113,10 +113,14 @@ type _StylesDesktopFalsy = _StylesDesktop | undefined | null | false
 export type StylesDesktop = _StylesDesktopFalsy | ReadonlyArray<_StylesDesktopFalsy>
 
 type _StylesMobileOverride = {
-  textAlignVertical?: 'auto' | 'top' | 'bottom' | 'center'
+  textAlignVertical?: 'top' | 'bottom' | 'center'
+  textAlign?: 'left' | 'right' | 'center' | 'justify'
```

Comment:
we remove `'auto'` as its incompatible with DOM and we never set it as its the default